### PR TITLE
feat: limit length title in highlight

### LINKED ISSUE #1192: Feature: Title needs to be limited to 49 characters.
### Type of feature

🍕 Feature

### Current behavior

Today the title for a highlight is unlimited.

ezgif com-video-to-gif (2)

### Suggested solution

Add a limit of 49 characters. If a highlight is submitted with a title longer than 49 characters, we should display a toast error message stating, "Title must be less than 49 characters."

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [ ] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by a0m0rajab**
## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [x] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description

Fixes https://github.com/open-sauced/insights/issues/1192
Fixes #1192 

## Related Tickets & Documents

related: https://github.com/open-sauced/insights/issues/1192

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

Not related but this is funny: 
Friday fire deploy

### CODE REVIEW

**Comment by a0m0rajab** [comment]
Side note: The compliance workflow does not work if the issue mentioned by link instead of number.

File: components/molecules/HighlightInput/highlight-input-form.tsx [hunk 1]
```diff
@@ -122,11 +122,12 @@ const HighlightInputForm = ({ refreshCallback }: HighlightInputFormProps): JSX.E
             <div className="flex pr-2">
               <input
                 value={title}
+                maxLength={49}
                 onChange={(e) => setTitle(e.target.value)}
                 className="flex-1 focus:outline-none"
                 type="text"
                 placeholder={
-                  isDivFocused ? "Add title (optional)" : "Click here to highlight your merged PRs and provide a link!"
+                  isDivFocused ? "Add title (optional, 49 char allowed)" : "Click here to highlight your merged PRs and provide a link!"
```
Comment (brandonroberts) [inline components/molecules/HighlightInput/highlight-input-form.tsx:130]: They will get a message on the form already if they go over the limit. We should make the `maxLength` 50 characters to match the API

```suggestion
                  isDivFocused ? "Add title (optional)" : "Click here to highlight your merged PRs and provide a link!"
```

**Review by brandonroberts — CHANGES_REQUESTED** [review]

File: components/molecules/HighlightInput/highlight-input-form.tsx (hunk 1 above)
Comment (a0m0rajab) [reply components/molecules/HighlightInput/highlight-input-form.tsx]: Done, Thanks you for your comment.

**Review by OgDev-01 — APPROVED** [review]
Great Job @a0m0rajab 🍕

**Review by brandonroberts — APPROVED** [review]

**Pull request merged**